app/models/stored_result.rb

```diff
@@ -0,0 +1,11 @@
+class StoredResult < ApplicationRecord
```

Comment:
Since this is kind of generic store that we can use elsewhere, should the name be a bit more generic? E.g. StoredRecord, StoredValue, DataBucket. I can imagine using it from storing user preferences, things like user cancelled tour for a given page etc. So the Result might be a bit misleading.

Reply:
Renamed to StoredValue, it seems as more generic name :) (not sure if not too generic, but we will see.